### DISCUSSION TITLE
Using HTML table in `.md` files instead of `.mdx`?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#785] [Q&A] [answered]
Hi,

Not sure if this is a bug so didn't report it in issues, but HTML tables such as

```html
<table border="1">
    <thead>
        <tr>
            <th>Header 1</th>
            <th>Header 2</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>Row 1, Cell 1</td>
            <td>Row 1, Cell 2</td>
        </tr>
        <tr>
            <td>Row 2, Cell 1</td>
            <td>Row 2, Cell 2</td>
        </tr>
    </tbody>
</table>
```

will not render in `.md` (it is invisible/no indication of text), but does in `.mdx`. Maybe this is the intended behaviour, just wanted to check since I know a lot of `.md` parsers (e.g. GitHub/GitLab flavoured) will render this correctly.

Ideally I'd want to keep files as `.md` unless there are specific `.mdx` features being used, but fully understand if the intention is not to support HTML tables in markdown.

P.s. similarly for iframes :D.

**Comment by fuma-nama** [comment, marked as answer]
to use HTML in markdown files, you are supposed to use MDX since we don’t render normal HTML but JSX (React components) instead. GitHub style table syntax (like the one mentioned in docs) is supported

**Reply by nktnet1** [reply]
Yep, understood. Thank you!

I needed a HTML table for multi-line text (without putting everything on one line in source code), but didn't want to use  `.mdx` file (wanted to communicate this to others) since everything else was markdown.

Not much of an issue to just use `.mdx` though - thanks for replying!
